lib/formulator.ex

```diff
@@ -2,31 +2,86 @@ defmodule Formulator do
   use Phoenix.HTML
   alias Formulator.HtmlError
 
+  @doc  """
+  Returns an html input with an associated label.
+  If there are errors associated with the field, it will also output a span
+  tag with the errors.
+
+  ## Options
+
+    * `:label` - when false, does not create a label tag. Instead, an `aria-label`
```

Comment:
I agree with the idea and I think I'm OK with making that move.

The reason I split them was for the case where you were intentionally wanting to provide more information to the aria-label such as in this case: https://github.com/thoughtbot/formulator/blob/d83d5be7d78c86b5b7e40783d00e778e7f41ad60/test/formulator_test.exs#L28

I think that's a pretty rare care so for now I'm going to follow the path you're recommending. When we get to the point that someone wants to customize the `aria-label` we can add that as another option.